tests/prophet/test_prophet_model_export.py

```diff
@@ -0,0 +1,414 @@
+):
+    mlflow.prophet.save_model(
+        pr_model=prophet_model.model, path=model_path, conda_env=prophet_custom_env
+    )
+
+    saved_pip_req_path = os.path.join(model_path, "requirements.txt")
+    _compare_conda_env_requirements(prophet_custom_env, saved_pip_req_path)
+
+
+def test_log_model_with_pip_requirements(prophet_model, tmpdir):
+    req_file = tmpdir.join("requirements.txt")
+    req_file.write("a")
+    with mlflow.start_run():
+        mlflow.prophet.log_model(prophet_model.model, "model", pip_requirements=req_file.strpath)
+        _assert_pip_requirements(mlflow.get_artifact_uri("model"), ["mlflow", "a"], strict=True)
+
+    # List of requirements
+    with mlflow.start_run():
+        mlflow.prophet.log_model(
+            prophet_model.model, "model", pip_requirements=[f"-r {req_file.strpath}", "b"]
+        )
+        _assert_pip_requirements(
+            mlflow.get_artifact_uri("model"), ["mlflow", "a", "b"], strict=True
+        )
+
+    # Constraints file
+    with mlflow.start_run():
+        mlflow.prophet.log_model(
+            prophet_model.model, "model", pip_requirements=[f"-c {req_file.strpath}", "b"]
+        )
+        _assert_pip_requirements(
+            mlflow.get_artifact_uri("model"),
+            ["mlflow", "b", "-c constraints.txt"],
+            ["a"],
+            strict=True,
+        )
+
+
+def test_log_model_with_extra_pip_requirements(prophet_model, tmpdir):
+    default_reqs = mlflow.prophet.get_default_pip_requirements()
+
+    # Path to a requirements file
+    req_file = tmpdir.join("requirements.txt")
+    req_file.write("a")
+    with mlflow.start_run():
+        mlflow.prophet.log_model(
+            prophet_model.model, "model", extra_pip_requirements=req_file.strpath
+        )
+        _assert_pip_requirements(mlflow.get_artifact_uri("model"), ["mlflow", *default_reqs, "a"])
+
+    # List of requirements
+    with mlflow.start_run():
+        mlflow.prophet.log_model(
+            prophet_model.model, "model", extra_pip_requirements=[f"-r {req_file.strpath}", "b"]
+        )
+        _assert_pip_requirements(
+            mlflow.get_artifact_uri("model"), ["mlflow", *default_reqs, "a", "b"]
+        )
+
+    # Constraints file
+    with mlflow.start_run():
+        mlflow.prophet.log_model(
+            prophet_model.model, "model", extra_pip_requirements=[f"-c {req_file.strpath}", "b"]
+        )
+        _assert_pip_requirements(
+            model_uri=mlflow.get_artifact_uri("model"),
+            requirements=["mlflow", *default_reqs, "b", "-c constraints.txt"],
+            constraints=["a"],
+            strict=False,
+        )
+
+
+@pytest.mark.small
+def test_model_save_without_specified_conda_env_uses_default_env_with_expected_dependencies(
+    prophet_model, model_path
+):
+    mlflow.prophet.save_model(prophet_model.model, model_path)
+    _assert_pip_requirements(model_path, mlflow.prophet.get_default_pip_requirements())
+
+
+@pytest.mark.small
```

Comment:
@BenWilson2 Can we mark these as large by removing `@pytest.mark.small`? I realize that there may be similar tests that are miscategorized as `small` in other flavors.

Reply:
certainly!